static/app/views/starfish/components/table.tsx

```diff
@@ -251,8 +251,16 @@ class _Table extends Component<Props, State> {
   ): React.ReactNode {
     const {eventView, location} = this.props;
 
-    const align = fieldAlignment(column.name, column.type, tableMeta);
-    const field = {field: column.name, width: column.width};
+    let align: Alignments = 'left';
+    if (column.column.kind === 'equation') {
+      align = 'right';
+    } else {
+      align = fieldAlignment(column.name, column.type, tableMeta);
+    }
+    const field = {
+      field: column.column.kind === 'equation' ? (column.key as string) : column.name,
+      width: column.width,
+    };
```

Comment:
we don't deal with equations well on this table ha, but don't want to spend time reworking that rn